src/libstd/io/stdio.rs

```diff
@@ -204,6 +204,30 @@ impl Stdin {
     ///
     /// For detailed semantics of this method, see the documentation on
     /// `BufRead::read_line`.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use std::io;
+    ///
+    /// #fn main() {
+    ///     let mut input = String::new();
+    ///     match io::stdin().read_line(&mut input) {
+    ///         Ok(n)      => {
+    ///             println!("{} characters read", n);
```

Comment:
Idiomatically we don't tend to put the extra spaces before the `=>` character, and the return value here is the number of bytes read, not the number of characters read.